.idea/compiler.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I assume this is an Android Studio setup.  If so, the commit name is misleading and should be something like "2. Setup Android Studio project".  This is an important step in the tutorial and it's not mentioned.